docker/recipes.test.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.4'
```

Comment:
The commit message for this one is misleading;  All actions can be done in non-fakemachine mode, what you probably mean is that it doesn't use actions which act on iage files which either needs fakemachine *or* access to loop devices